sklearn/preprocessing/tests/test_data.py

```diff
@@ -149,12 +168,14 @@ def test_standard_scaler_numerical_stability():
 def test_scaler_2d_arrays():
     # Test scaling of 2d array along first axis
     rng = np.random.RandomState(0)
-    X = rng.randn(4, 5)
+    n_samples = 4
+    X = rng.randn(n_samples, 5)
```

Comment:
You can also define `n_features = 5` and reuse it to make the assertion on `n_features * [0.0]` more explicit.